example/stacks/terraform-backend.yaml

```diff
@@ -0,0 +1,61 @@
+type: s3
```

Comment:
Hrmm... `type` at the top-level sounds too generic. 

What if we had:

```
terraform:
  remote_backend: s3
  backend:
    s3:
```

Reply:
will improve that in the next PR